.github/auto-assignees.yml

```diff
@@ -9,7 +9,6 @@ reviewers:
 
   groups:
     maintainers:
-      - zubron
```

Comment:
No idea why removing `zubron`. It would be more clear if you could leave any comments about this.

Reply:
Bridget(`zubron`) isn't the maintainer anymore and the `auto-assignees` Github Action task which assigns reviewers automatically reads the reviewers from the list and fails if the reviewer isn't a maintainer.

BTW, `zubron` is already removed from the list by PR https://github.com/vmware-tanzu/velero/pull/4749

Reply:
I see. Thanks for your explanation.

This is not a block comment. But still, feel confused about this change. `zubron` already not exist in the `main` branch.